qa/tasks/cephfs/fuse_mount.py

```diff
@@ -405,16 +405,17 @@ def _asok_path(self):
     def _prefix(self):
         return ""
 
-    def admin_socket(self, args):
+    def find_admin_socket(self):
         pyscript = """
 import glob
 import re
 import os
 import subprocess
 
-def find_socket(client_name):
+def _find_socket(client_name):
```

Comment:
I don't understand why we're changing this method. Please break out into a separate commit with an explanation.

Reply:
Done.v